builtin/logical/ssh/path_config_issuers.go

```diff
@@ -0,0 +1,139 @@
+package ssh
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/openbao/openbao/sdk/v2/framework"
+	"github.com/openbao/openbao/sdk/v2/logical"
+)
+
+func pathConfigIssuers(b *backend) *framework.Path {
+	configIssuerSchema := map[int][]framework.Response{
+		http.StatusOK: {{
+			Description: "OK",
+			Fields: map[string]*framework.FieldSchema{
+				"default": {
+					Type:        framework.TypeString,
+					Description: `Reference (name or identifier) to the default issuer.`,
+					Required:    true,
+				},
+			},
```

Comment:
I forget, did we talk about `default_follows_latest_issuer` and what we felt about that?

Reply:
Yes, we did. I don't quite remember the details, but I believe we agreed that it was unnecessary as there's an endpoint to submit issuers with optional names. Let me know what you think, it's also possible that I have extrapolated this out of some discussion we had.

Reply:
Nope, you're right. There's no hierarchy and the import-with-name handles most of the remaining use case here.